packages/a/aws-c-common/xmake.lua

```diff
@@ -0,0 +1,26 @@
+package("aws-c-common")
+    set_homepage("https://github.com/awslabs/aws-c-common")
+    set_description("Core c99 package for AWS SDK for C")
+    set_license("Apache-2.0")
+
+    add_urls("https://github.com/awslabs/aws-c-common/archive/$(version).tar.gz")
+    add_versions("v0.6.9", "928a3e36f24d1ee46f9eec360ec5cebfe8b9b8994fe39d4fa74ff51aebb12717")
+
+    if is_plat("linux") then
+        add_syslinks("dl", "m", "pthread", "rt")
+    end
+
+    on_install("linux", function (package)
+        local shared = package:config("shared")
+        local configs = {}
+
+        -- Build type
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
+
+        -- Configs
+        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (shared and "ON" or "OFF"))
```

Comment:
package:config("shared") and "ON" or "OFF"

Reply:
I'm using the `shared` local variable declared above. Isn't that good? Am I missing something?

Reply:
If it is only used in this line of code, there is no need to define additional variables.